mx.truffleruby/mx_truffleruby.py

```diff
@@ -57,16 +58,34 @@ def getResults(self):
 class TruffleRubyLauncherBuildTask(mx.ArchivableBuildTask):
     def __init__(self, *args):
         mx.ArchivableBuildTask.__init__(self, *args)
+
+        self.jvm_args_file = join(root, 'tool', 'jvm_args')
+        self.mx_env = join(root, 'mx.truffleruby', 'env')
+        self.suite_py = join(root, 'mx.truffleruby', 'suite.py')
```

Comment:
@gilles-duboscq @dougxc Is there a better way to get those paths from mx or this is fine?

Reply:
There is a `suite_py()` method on `Suite` and a `mxDir` property on `Suite` to get to `<trufflerubysuite>/mx.truffleruby`

Reply:
Thanks!